docs/getting-started/introduction.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Introduction
+---
+
+# Introduction
+
+## What is EventStoreDB?
+
+EventStoreDB is an event-native database designed specifically to store, process, and deliver application state changes, known as events.
+
+Each event captures a specific change in an application’s state, like placing an order, processing a payment, or shipping an order. Capturing distinct, incremental update in a business process.
+
+EventStoreDB offers the following features:
+
+| Feature | Description |
+|---------|-------------|
+| Append-only Event Log | Audit trail that captures all historical changes as events for one or more systems. |
```

Comment:
"Audit trail" light be misleading .
Not sure what other words to use though.

Reply:
I hear you .. will look for something.

Reply:
Will be changing it to:

`A durable, sequential, and immutable log that captures events in a consistent order.`